app/components/AppToolbar/index.js

```diff
@@ -30,6 +30,7 @@ const styleSheet = createStyleSheet('zmoraAppToolbar', (theme) => ({
   toolbar: {
     transform: 'translate3d(0,0,0)',
     overflow: 'hidden',
+    backgroundColor: '#673BB7',
```

Comment:
@maxmati @marcmos czy kolor AppBara ustawia się za pomocą jakichś globalnych primary kolorów jak w androidzie czy np. w ten sposób? A jak za pomocą primary to gdzie się go ustawia